src/lib/files.ts

```diff
@@ -9,3 +9,30 @@ export const readAsText = (file: File): Promise<string> => {
     reader.readAsText(file)
   })
 }
+
+export const getFilesFromClipboard = (event: ClipboardEvent) => {
+  const files = event.clipboardData!.files
+  const images: File[] = []
+
+  if (files instanceof FileList) {
+    for (let i = 0; i < files.length; i++) {
+      const file = files[i]
+      const newName = String(file.lastModified)
+      const newFile = renameFile(file, newName)
```

Comment:
We might need to discuss more about file naming here? How should we generate a name for the attachment? @Rokt33r

Reply:
Since all the screenshots, at least in Linux, have the same name `image.png`, I thought that using the timestamp for the name was a good idea, but yes, it can be rethought.